test/unit/http_test.lua

```diff
@@ -242,6 +244,35 @@ local test_mixed_cases = {
             "http://127.0.0.1:8081/metrics4",
         },
     },
+    ["listen_tls"] = {
+        cfg = {
```

Comment:
In such simple tests we can check more parameters. Preferably all of them.

Please, add test cases with `ciphers`, `password`, `password_file`.

Reply:
Moved all cases into separate test set.